ingest/raw_compressed_array_schema.json

```diff
@@ -0,0 +1,14 @@
+[
+{
+   "description": "[DESCRIPTION]",
+   "name": "basic_array_data",
+   "type": "Integer",
+   "mode": "Nullable"
+ },
+ {
+   "description": "[DESCRIPTION]",
+   "name": "raw_array_data",
+   "type": "Integer",
+   "mode": "Nullable"
+ }
+]
```

Comment:
should at least the first if not both be "Required"?

Reply:
Oh, good point. I'll make `basic_array_data` required and I think I should do the same for `raw_array_data` but I'm not sure. Is there ever a case where we wouldn't have raw_array_data? Would our scripts just output a 0 for that field in that case?